src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTMetaSearch.java

```diff
@@ -58,10 +54,10 @@ private DataGroup  getDataFromURL(TableServerRequest request) throws Exception {
         }
 
         //use fully specified path, which includes logical database, schema, and table name
-        //ex. URL http://lsst-qserv-dax01:5000/meta/v1/db/W13_sdss_v2/sdss_stripe82_01/tables/RunDeepForcedSource
+        //ex. URL http://lsst-qserv-dax01:5000/meta/v1/db/W13_sdss_v2/sdss_stripe82_01/tables/RunDeepForcedSource/
         String url = LSSTQuery.getMetaservURL() + URLEncoder.encode(parts[0], "UTF-8") + "/" +
                 URLEncoder.encode(parts[1], "UTF-8") + "/tables/"  +
-                URLEncoder.encode(parts[2], "UTF-8");
+                URLEncoder.encode(parts[2], "UTF-8") + "/";
```

Comment:
Hmm.. haven't thought about needing to encode url path.  This rely on the resource provider doing the same.
Are you sure it's needed?

Reply:
You are right. We should not use URLEncoder for path segments, only for query string. The set of reserved characters is different for path segments and query parameter values is different. I vaguely remember path segments like `"//lsst-qserv-master01:4040"` at some point (a workaround for missing schema name). URLEncoder was good enough for this.